db/re/mob_db.conf

```diff
@@ -65147,15 +65147,441 @@ mob_db: (
 		Amdarais_Card: 1
 	}
 },
-//2477,NG_WANDER_MAN
-//2478,NG_RIDEWORD
-//2479,NG_MIMIC
-//2480,NG_EVIL_DRUID
-//2481,NG_WRAITH_DEAD
-//2482,G_MG_KHALITZBURG
-//2483,NG_BAPHOMET
-//2484,G_NG_BAPHOMET_
-//2485,NG_CHIMERA
+// Nightmare Glast Heim
+{
+	Id: 2477
+	SpriteName: "NG_WANDER_MAN"
+	Name: "Nightmare Wander Man"
+	Lv: 151
+	Hp: 130682
+	Sp: 1
+	Exp: 19814
+	JExp: 17228
+	AttackRange: 2
+	Attack: [2827, 3045]
```

Comment:
i think for the monster attack is should be [2827, 218]  like other monsters the attack2-attack1 in divine-pride